### ISSUE TITLE
fermion device receives poor json

### ISSUE THREAD

**Issue by fretchen** [synqs/pennylane-ls#32]
Running

```
@qml.qnode(FermionDevice)
def simple_hopping(theta = 0):
    '''
    The circuit that simulates the experiments.
    
    theta ... angle of the hopping
    '''
    # load atoms
    FermionOps.Load(wires=0)
    FermionOps.Load(wires=1)
    
    # let them hop
    FermionOps.Hop(theta, wires=[0,1,2,3])

    # measure the occupation on the right side
    obs = FermionOps.ParticleNumber(0) @ FermionOps.ParticleNumber(1)
    return qml.expval(obs)
```
Leads too:

`{"job_id": 487, "status": "INITIALIZING", "detail": "Got your json."}`

So we will have to change the way these devices work with redis and most likely add a waiting time.

**Comment by TheNewYorkerFlute** [comment]
comment of Rohit: Wait for some time after submission and before asking for the result. This is specific for redis and should not be a problem with the current backend.

**Comment by fretchen** [comment]
So we will have to fix this in the future as the real machines will make the same thing right ?

**Comment by fretchen** [comment]
This should not appear anymore as long as there is no queuing happening in the back-end. However, if the back-end `http://qsim-redis.herokuapp.com/` is selected I expect the bug to reappear. Or once this is merged into `qsimsim`...

**Comment by fretchen** [comment]
I think that this is related to #11
